metastore/src/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -228,13 +228,10 @@ static public void setCmRoot(Path cmRoot) {
    *   to a deterministic location of cmroot. So user can retrieve the file back
    *   with the original location plus checksum.
    * @param conf
-   * @param checkSum checksum of the file, can be retrieved by {@link getCksumString}
-   * @return
-   * @throws IOException
-   * @throws MetaException
+   * @param checkSum checksum of the file, can be retrieved by {@link #checksumFor(Path, FileSystem)}
+   * @return Path
    */
-  static Path getCMPath(Configuration conf, String checkSum)
-      throws IOException, MetaException {
+  static Path getCMPath(Configuration conf, String checkSum) throws IOException, MetaException {
```

Comment:
@sankarh unfortunately i was not confident about the change, as I had just formatted the code and requires very less diligence on my side, but i have since then, looked back in it and made the requested  modifications from you. I am waiting for apache jira to come back to attach the new patch